### ISSUE TITLE
Use __func__ on C99+?

### ISSUE THREAD

**Issue by aartaka** [J-M0/dbg.h#2]
I see you're using `__FILE__` and `__LINE__` in your macros, but I don't see a `__func__` (added in C99). Is there any reason not to use it? I also see C11 `_Generic` in your code, so it's definitely not portability reasons.

**Comment by J-M0** [comment]
Nope, there wasn't any particular reason I didn't use `__func__`. I'll consider it and see if I like how it looks in the output.

**Comment by aartaka** [comment]
Well, more metadata is always better, I guess :shrug:
